base/missing.jl

```diff
@@ -135,7 +135,7 @@ round(::Type{T}, ::Missing, ::RoundingMode=RoundNearest) where {T} =
     throw(MissingException("cannot convert a missing value to type $T: use Union{$T, Missing} instead"))
 round(::Type{T}, x::Any, r::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype_checked(T), x, r)
 # to fix ambiguities
-round(::Type{T}, x::Rational, r::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype_checked(T), x, r)
+round(::Type{T}, x::Rational{Tr}, r::RoundingMode=RoundNearest) where {T>:Missing,Tr} = round(nonmissingtype_checked(T), x, r)
```

Comment:
Why add the parameter `Tr` here? It isn't used?

Reply:
Without the added parameter, the method ambiguity tests fail, see https://build.julialang.org/#/builders/22/builds/8886

I was surprised by this, too. Since there are no restrictions on the type parameter, I thought these methods would be equally specific. Does it make sense to open an issue for this?

Reply:
Hmm, that is odd. No idea what is going on there.